### ISSUE TITLE
cairomm binaries have checksum mismatch

### ISSUE THREAD

**Issue by superloach** [skycocker/chromebrew#2589]
Arch: x86_64
```
$ crew install cairomm
cairomm: The Cairomm package provides a C++ interface to Cairo.
https://www.cairographics.org/
version 1.12.2
Precompiled binary available, downloading...
Checksum mismatch. :/ Try again.
```

**Comment by superloach** [comment]
It appears that there is something wrong with the Bintray URL.
```
$ wget https://dl.bintray.com/chromebrew/chromebrew/cairomm-1.12.2-chromeos-i686.tar.xz
--2018-08-22 14:07:24--  https://dl.bintray.com/chromebrew/chromebrew/cairomm-1.12.2-chromeos-i686.tar.xz
Resolving dl.bintray.com... 35.160.90.8, 52.39.244.1
Connecting to dl.bintray.com|35.160.90.8|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-08-22 14:07:25 ERROR 404: Not Found.
```

**Comment by superloach** [comment]
Uhh, why did you add the i686 label @uberhacker? I'm on x86_64 😅

**Comment by superloach** [comment]
I was testing all of the Bintray URLs, and just copied the error from the i686 one at random. They're all 404-ing.

**Comment by superloach** [comment]
This has been temporarily fixed by commenting out the binaries. #2598

**Comment by JL2210** [comment]
This can be closed.

**Comment by superloach** [comment]
I feel that this is still an issue until functional binaries are remade.

**Comment by JL2210** [comment]
I'll try it.

**Comment by uberhacker** [comment]
Fixed in #2823.
